Add Astrian/tg-watchdog (#24)
grammyjs/awesome-grammY: README.md

- [bot-base/scan-tool-bot](https://github.com/bot-base/scan-tool-bot) - Scan QR codes with your camera. [ᴜsᴇ](https://t.me/ScanToolBot)
- [ShinkarenkoMaxim/simple-quiz](https://github.com/ShinkarenkoMaxim/simple-quiz) - A simple quiz bot as Web App. [ᴜsᴇ](https://t.me/yaqb_bot)
- [dcdunkan/show-json-bot](https://github.com/dcdunkan/show-json-bot) - See the JSON data behind Telegram messages. [ᴜsᴇ](https://t.me/jsoonbot)

> - [Astrian/tg-watchdog](https://github.com/Astrian/tg-watchdog) - The brand-new solution for anti-abuse from Telegram groupchats. [ᴜsᴇ](https://t.me/WatchdogVerifyBot?startgroup=start&admin=can_invite_users)

Reviewer: ```suggestion
- [Astrian/tg-watchdog](https://github.com/Astrian/tg-watchdog) - Anti-abuse solution for Telegram groups. [ᴜsᴇ](https://t.me/WatchdogVerifyBot?startgroup=start&admin=can_invite_users)
```

> - [Astrian/tg-watchdog](https://github.com/Astrian/tg-watchdog) - The brand-new solution for anti-abuse from Telegram groupchats. [ᴜsᴇ](https://t.me/WatchdogVerifyBot?startgroup=start&admin=can_invite_users)

Reviewer: `can_invite_users`?

Author: It suggests the Telegram app request the user to add the bot to the group and select `can_invite_user` permission as default.

Reviewer: Why would it need that permission?

Author: This permission is related to Join Requests, which allow the hot to receive alerts of new Join Requests, chat with candidates actively, and approve or decline their requests. And that's the only permission the bot need.

Detail: https://core.telegram.org/bots/api-changelog#december-7-2021
